aztec-nargo/Earthfile

```diff
@@ -15,3 +15,9 @@ run:
     ENV PATH "/usr/bin:${PATH}"
     ENTRYPOINT ["/usr/bin/tini", "--", "/usr/bin/compile_then_transpile.sh"]
     SAVE IMAGE aztecprotocol/aztec-nargo
+
+export-aztec-nargo:
```

Comment:
Ahh, does this mean we'll soon be able to push aztec-nargo from Github Actions instead of CircleCI?

Reply:
Should be able to yes.